csrc/scheduler/normalization_inner_outer_tma_ws.cpp

```diff
@@ -307,8 +218,7 @@ void getHeuristics(
             << "multiple_reds_per_blk: " << rparams->multiple_reds_per_blk
             << "\n"
             << "warps_per_sm: " << iop.warps_per_sm << "\n"
-            << "gdimy: " << iop.gdimy << "\n"
-            << "block(" << (iop.bdimx) << ", " << iop.bdimy << ", " << 1 << ")";
+            << "gdimy: " << iop.gdimy << "\n";
```

Comment:
we probably still want to print out `iop.bdimx` ?

Reply:
added